linkedList.go

```diff
@@ -34,30 +40,38 @@ func (l *L) Find(n int) (index uint, found bool) {
 	return 0, false
 }
 
-func (l *L) Get(index uint) (int, bool) {
-	current := l.head
+func (l *LinkedList) Get(index uint) (int, bool) {
+	l.mutex.Lock()
```

Comment:
I think it is better to handle the locking outside of the library so that the user can choose to use locking if they need.